app/renderer/prefilledWallet.ts

```diff
@@ -0,0 +1,84 @@
+import * as crypto from "crypto"
+import {
+  Wallet,
+  Account,
+  FinalKey,
+  CURRENT_WALLET_VERSION
+} from "app/common/runtimeTypes/storage/wallets"
+
+export const prefilledWalletCaption = "" // TODO(anler): This needs to be defined
+
+export const prefilledWallet = generatePrefilledWallet()
+
+/**
+ * Internal function that generates a pre-filled wallet. Do not use outside of this module.
+ */
+function generatePrefilledWallet(): Wallet {
+  const chainCode = Buffer.from(
+    [152, 8, 159, 246, 221, 27, 159, 171, 139, 98, 82, 232, 229, 140, 201, 2, 78, 113, 141, 104,
+      187, 192, 191, 147, 91, 108, 192, 174, 214, 251, 100, 79])
+  const masterSecret = Buffer.from(
+    [196, 188, 49, 56, 180, 80, 166, 118, 177, 247, 66, 74, 173, 113, 48, 127, 194, 4, 164, 0,
+      70, 14, 21, 211, 113, 209, 238, 121, 124, 251, 139, 168])
+
+  const id = crypto.pbkdf2Sync(
+    Buffer.concat([chainCode, masterSecret]),
+    "sheikah seed", 4096, 32, "sha256"
+  ).toString("hex")
+  const _v = CURRENT_WALLET_VERSION
+  const caption = "My prefilled wallet"
+  const epochs = {}
+
+  const seed = { seed: { chainCode, masterSecret }, kind: ("Wip3" as "Wip3") }
+
+  const accounts = [generatePrefilledAccount()]
+
+  return { id, _v, caption, seed, epochs, purpose: 0x80000003, accounts }
+}
+
+/**
+ * Internal function that generates a pre-filled account. Do not use outside of this module.
+ */
+function generatePrefilledAccount(): Account {
+  const balance = 4.24
+  const keyPath = [2147483651, 2147488567, 2147483648]
+  const finalKey = generateFinalKey(keyPath, 0)
+  const keyChains = [
+    {
+      keyPath: [...keyPath, 0],
+      finalKeys: [finalKey],
+    },
+    {
+      keyPath: [...keyPath, 1],
+      finalKeys: [finalKey],
+    },
+    {
+      keyPath: [...keyPath, 2],
+      finalKeys: [finalKey],
+    }
```

Comment:
Perhaps, it would be better to call `generateFinalKey(keyPath, index)` with the proper index value.